warehouse/query-core/src/test/java/datawave/query/util/WiseGuysIngest.java

```diff
@@ -1098,7 +1098,7 @@ private static Value getValueForBuilderFor(String... in) {
      */
     private static Value getValueForNuthinAndYourHitsForFree() {
         Uid.List.Builder builder = Uid.List.newBuilder();
-        builder.setCOUNT(50); // better not be zero!!!!
+        builder.setCOUNT(0); // better not be zero!!!!
```

Comment:
can you update the comment here to indicate why we are reducing to zero?

Reply:
sure